packages/playground/CoreApp/App.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
can you clarify why? the reason for putting it here is to a) prove that it works, and reuse the setup for RNTester (just like playground-win32 / playground-win32 (packaged) already do)

Reply:
What is the long term use of putting this here?

Playground is our ad-hoc testing app, which _can_ load the RNTester JS, but also loads other sample bundles.

E2ETest app is our e2etest app, which also loads the RNTester's JS, to run tests against it.

Using the RNTester's JS isn't I think enough justification to put a completely independent application as a subfolder within playground, when it has nothing to do with the playground package and what it provides.

Reply:
At least open an issue tracking to replace this with a proper sample package folder under `packages` (whether it's independent like I've suggested, or because you've updated one or more of our other packages (e2etest, sample-apps, etc.) to this style of app)

Then there should be a similar issue to track updating the new app template.

Reply:
filed #10283